.gitignore

```diff
@@ -4,6 +4,7 @@ node-jiti
 
 # dependencies
 node_modules
+package-lock.json
```

Comment:
This is not necessary.
lock file is not included in the diff if the library is not added.